src/boards/include/boards/olimex_rp2350_xl.h

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2020 Raspberry Pi (Trading) Ltd.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+// -----------------------------------------------------
+// NOTE: THIS HEADER IS ALSO INCLUDED BY ASSEMBLER SO
+//       SHOULD ONLY CONSIST OF PREPROCESSOR DIRECTIVES
+// -----------------------------------------------------
+
+// This header may be included by other board headers as "boards/olimex_rp2350_xl.h"
+
+#ifndef _BOARDS_OLIMEX_PICO2_XL
```

Comment:
For consistency with the filename, please change this to `_BOARDS_OLIMEX_RP2350_XL_H` (alternatively, I guess you could rename the file?)